src/components/common/__snapshots__/LoadingPage.test.js.snap

```diff
@@ -1,5 +1,23 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`LoadingPage should handle showHelpText prop properly 1`] = `
+<div
+  className="Loading"
+>
+  <div
+    className="Loading-title"
+  >
+    Loading
+  </div>
+  <Loader
+    color="#171124"
+    css=""
+    loading={true}
+    size={250}
```

Comment:
Tests failing on my system due to the snapshot:

![image](https://user-images.githubusercontent.com/7704542/75100573-331ffd80-5584-11ea-8d80-755b75ff5207.png)

Reply:
When's the last time you've updated your `node_modules`? I believe that we updated `react-spinners` which changes the props that `<Loader>` exposes. The test passes on both my local computer and the CI, so not exactly sure what's happening?